Sources/MongoSwift/BSON/BSONDocument.swift

```diff
@@ -300,7 +300,7 @@ extension BSONDocument {
 
     /// Returns the relaxed extended JSON representation of this `BSONDocument`.
     /// On error, an empty string will be returned.
-    public var extendedJSON: String {
+    public func toExtendedJSONString() -> String {
```

Comment:
I was going to say it would be better to throw than to return an empty string now that these are methods, but OTOH i think in the new library we don't expect JSON serialization to fail unless it's explicitly unvalidated BSON. in which case i think fatal error / empty string is acceptable